src/index.tsx

```diff
@@ -14,7 +14,6 @@ import { TextEncoder } from 'text-encoding';
 
 import { localAuthStorageKey } from 'constants/localAuth';
 
-import 'uswds';
```

Comment:
I don't think we want to use any of the Javascript provided by USWDS since we and `react-uswds` provide that functionality.

This import was colliding with the `ComboBox` so I removed it.